index.js

```diff
@@ -794,6 +798,14 @@ SendStream.prototype.stream = function stream (path, options) {
 
   // pipe
   var stream = fs.createReadStream(path, options)
+  if (this._transform) {
+    // When using a transform stream, the implementor is
+    // responsible for setting content length or any other
+    // headers for their stream type.
+    res.removeHeader('Content-Length')
+
+    stream = this._transform(stream, res, options)
```

Comment:
So since the original `stream` is overwritten and now lost; does this open open the floor to fd leaks again? Does using a transform stream mean users now need to be careful not to cause them with the fs steam encounters an error, or similar?